### PR TITLE
Fix PB search for stored boolean fields

### PR DESCRIPTION
**Pull request by rzezeski**
Addresses #209.
- Fix encoding function so that it treats boolean fields properly.
- Add test to verify retrieval of string, float, and boolean fields via PB search.
- Update the BB driver to enabled benchmarking of PB search.

### CODE REVIEW

File: src/yz_pb_search.erl [hunk 1]
```diff
@@ -140,17 +140,33 @@ default([H|T], _) ->
...
+
+%% @private
+%%
+%% NOTE: Need to do this here because `riak_pb_codec' doesn't convert
+%% numbers.
+-spec encode_val(number() | atom() | binary()) -> atom() | binary().
+encode_val(Val) when is_integer(Val) ->
+    integer_to_binary(Val);
+encode_val(Val) when is_float(Val) ->
+    float_to_binary(Val);
```
Comment (coderoshi) [inline src/yz_pb_search.erl:169]: integer_to_binary and float_to_binary don't exist before R16

**Comment by coderoshi** [comment]
Search passes +1

**Comment by rzezeski** [comment]
If we go back to 15B0x or a user absolutely needs to run Yokozuna on it then we'll patch these calls. But for now going to revel in 16B02 :)

File: src/yz_pb_search.erl (hunk 1 above)
Comment (jaredmorrow) [inline src/yz_pb_search.erl:169]: I also got bit by this trying to see if we could still build on R15B01.

**Pull request merged**